src/components/users/signin-social/ko/runtime/signin-aad-b2c.ts

```diff
@@ -60,15 +63,22 @@ export class SignInAadB2C {
             }
             await this.selectedService.checkCallbacks();
         }
+        else {
+            this.logger.trackEvent(eventTypes.aadB2CLogin, { message: "AAD B2C client configuration is missing." });
+        }
+
+        this.logger.trackEvent(eventTypes.aadB2CLogin, { message: "Signin AAD B2C component initialized." });
```

Comment:
Shouldn't this event be tracked inside the
`if (this.aadConfig) {` when the config actually presents?  Otherwise in case when the configs are not presenting which is an error scenario, but the logs will report `component initialized`.

Reply:
I'm not sure.. Ideally, we shouldn't kick off the initialization of this widget if the config isn't present. However, in this situation, even if we somehow initiated the process without a config, it would still go through. It's an unusual scenario, but as long as it doesn't trigger an error, the initialization can wrap up fine.
In this case, I think it's important to have both logs, so we know that the initialization was completed without the config.

I wouldn't treat t as an error scenario, because there was no error thrown. Maybe we could treat it as a warning since indeed the widget doesn't make any sense without the config (though, at the moment, we don't have support for different log levels from the client side). What are your thoughts on this?

Reply:
If we are referring to the component overall yes I do agree that initialized at the end is a good information! :) Thank you